### ISSUE TITLE
Added rule, but breach was not detected

### ISSUE THREAD

**Issue by Synesso** [woltapp/arrow-detekt-rules#11]
I included the plugin
```
subprojects {
  apply(plugin = "io.gitlab.arturbosch.detekt")
  apply(plugin = "java")
  apply(plugin = "kotlin")

  detekt {
    toolVersion = "1.22.0"
    config = files("${rootDir}/.detekt.yml")
    buildUponDefaultConfig = true
  }

  dependencies {
    detektPlugins("com.wolt.arrow.detekt:rules:0.1.0")
  }

...
```

configured it:
```
ArrowRuleSet:
  NoEffectScopeBindableValueAsStatement:
    active: true
```

Tried to force an error

```kotlin
either {
  eitherOne() // .bind()
  eitherTwo().bind()
  eitherThree().bind()
}
```

Ran `gradle detekt`.

It was successful where I expected it to fail.

I know that the configuration of the plugin is correct because if I move the `ArrowRuleSet` declaration to some other scope in the detekt config it will throw a parse error.

**Comment by diastremskii** [comment]
Hey, thanks for the report!
This is one of the Detekt rules that require [type resolution](https://detekt.dev/docs/gettingstarted/type-resolution/) and by default, Detekt runs without it. Could you please test with `gradle detektMain` (for main source set) or `gradlew detektTest` (for test source set)? That would run Detekt with type resolution enabled and the rules that require it should be applied.

**Comment by Synesso** [comment]
Thank you. This is news to me. I thought I had gotten to zero issues, but running with type resolution has uncovered ~800 "issues". Looks like I have more work to do...
